### PR TITLE
Remove sort bar in note lists and trash

### LINKED ISSUE #1342: Remove Sort Bar
Based on low usage, let's remove the sort bar from both the search results and notes list.

Sorting options should now appear in Settings only.

### PR DESCRIPTION
**Pull request by danilo04**
Fixes #1342 #1358

### Fix
Removes the sort bar from both the search results and notes list.

### Test

1. Login with a valid account
2. Go to note list. The sort bar should not appear
3. Go to Trash list and the sort bar should not appear either
4. Go the Settings and change the sort settings
5. Come back to the note list and the notes should be sorted based on the setting set

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
`RELEASE-NOTES.txt` was updated in 0eeacf0cf885231360849062e2cb97afe72c7cf0 with:
> Removed sort bar from notes list but still accessible from settings

### CODE REVIEW

**Comment by SylvesterWilmott** [comment]
Looks good. Let's just add to the release notes that the sort bar was removed but is still accessible from the settings.

File: Simplenote/src/main/java/com/automattic/simplenote/NoteListFragment.java [hunk 1]
```diff
@@ -470,16 +406,6 @@ public boolean onMenuItemClick(MenuItem item) {
         getListView().setMultiChoiceModeListener(this);
     }
 
-    private void updateSortOrder(String label, int index) {
-        AnalyticsTracker.track(
-            AnalyticsTracker.Stat.LIST_SORTBAR_MODE_CHANGED,
```
Comment (ParaskP7) [inline Simplenote/src/main/java/com/automattic/simplenote/NoteListFragment.java:475]: Minor (🔍): Consider removing this unused tracking `Stat` as well: `LIST_SORTBAR_MODE_CHANGED`

**Review by ParaskP7 — APPROVED** [review]
👋 @danilo04 !

I have reviewed and tested this PR as per the instructions, everything works as expected, good job! 🌟 

I have left one minor (🔍) comments for you to consider. I am going to approve this PR anyway, since none is blocking. I am **NOT** going to merge this PR yet to give you some time to apply any of my suggestions. However, feel free to ignore them and merge the PR yourself.

File: Simplenote/src/main/java/com/automattic/simplenote/NoteListFragment.java (hunk 1 above)
Comment (danilo04) [reply Simplenote/src/main/java/com/automattic/simplenote/NoteListFragment.java:475]: Good catch! Removed in 8f10c597c69df02abe3f343f433a70bbd47e9500.

**Comment by danilo04** [comment]
Hi @SylvesterWilmott. Thanks for the review. Is it OK the following:

> Removed sort bar from notes list but still accessible from settings

**Comment by danilo04** [comment]
Thanks for the review @ParaskP7!

File: Simplenote/src/main/java/com/automattic/simplenote/NoteListFragment.java (hunk 1 above)
Comment (ParaskP7) [reply Simplenote/src/main/java/com/automattic/simplenote/NoteListFragment.java:475]: Perfecto! 🥇

**Review by ParaskP7 — APPROVED** [review]
👋 @danilo04 !

If nothing was changed since my last review, it is still a 🟢  light from my side, feel free to merge this PR when you feel ready.

🙏

**Pull request merged**